src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBER_OF_FIGURES = 5;
+    private static final int MAX_SIDE = 100;
+    private static final int MAX_WIDTH = 200;
+    private static final int MAX_HEIGHT = 250;
+    private static final int MAX_BASE = 50;
+    private static final int MAX_RADIUS = 25;
+    private static final int MAX_TOP_BASE = 50;
+    private static final int MAX_BOTTOM_BASE = 100;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+    private int randomSide;
+    private int randomBase;
+    private int randomWidth;
+    private int randomHeight;
+    private int randomRadius;
+    private int randomTopBase;
+    private int randomBottomBase;
+
+    public Figure getRandomFigure() {
+        Figure randomFigure = null;
+        Color randomColor = Color.valueOf(colorSupplier.getRandomColor());

```

Comment:
I know, by task condition I have to return String from getRandomColor, but I use Enum Color in every constructor and I think it's easier return Color to avoid redandant transformation. Could you give me advise how to solve this better?